training/supres_train.py

```diff
@@ -138,9 +138,11 @@ def on_epoch_end(self, epoch, logs=None):
         action="store_true",
         help="Whether to run a 60->10m network. Default 20->10m.",
     )
-    parser.add_argument("--deep", action="store_true", help=".")
     parser.add_argument(
-        "--aesr", action="store_true", help="Use AutoEnconder SR network"
+        "--model",
+        default="dsen2",
+        choices=["vdsen2", "aesr", "srcnn", "rednet", "resnet"],
```

Comment:
What is vdsen2? Was that part of the original repo?

Reply:
Yes - it's a "very deep" DSEN2 model, instead of having 6 layers as in DSEN2, we use 32.